pennylane_lightning/core/src/utils/TestHelpersWires.hpp

```diff
@@ -53,6 +53,30 @@ inline auto createWires(Pennylane::Gates::GateOperation op, size_t num_qubits)
     return {};
 }
 
+inline auto createWires(Pennylane::Gates::ControlledGateOperation op,
+                        [[maybe_unused]] size_t num_qubits)
+    -> std::vector<size_t> {
+    // TODO: needed when introducing C(MultiRZ)
+    // if (array_has_elem(Pennylane::Gates::Constant::multi_qubit_gates, op)) {
+    //     std::vector<size_t> wires(num_qubits);
+    //     std::iota(wires.begin(), wires.end(), 0);
+    //     return wires;
+    // }
```

Comment:
Is this block needed?

Reply:
This is preparation for #614 . I would like to uncomment it once that gets merged.